src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -115,7 +114,7 @@ object DockerPlugin extends AutoPlugin {
         case _       =>
       }
 
-      val oldFunction = (dockerLayerGrouping in Docker).value
+      val oldFunction = (Docker / dockerLayerGrouping).value
```

Comment:
```suggestion
      val oldFunction = dockerLayerGrouping.value
```

This should be removed here then as well